src/context/AppProvider/AppContext.tsx

```diff
@@ -168,6 +173,19 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
       dispatch(getValidatorData.initiate(accountId, options))
     }
 
+    dispatch(
+      getOpportunityMetadata.initiate(
```

Comment:
Grep for `getFoxEthLpMetrics.initiate` and initiate this query everywhere we currently fire the `foxEthLpMetrics` query.

We might also want to dispatch it `onMessage` so we react on incoming Txs